config/src/config/indexer_grpc_config.rs

```diff
@@ -42,9 +42,14 @@ pub struct IndexerGrpcConfig {
     /// Number of transactions returned in a single stream response
     pub output_batch_size: u16,
 
+    #[serde(default = "default_enable_verbose_logging")]
     pub enable_verbose_logging: bool,
 }
 
+fn default_enable_verbose_logging() -> bool {
+    false
+}
+
```

Comment:
It's really dangerous to have multiple levels of defaults. You will easily get lost in which value is used where.